lib/Conversion/TritonGPUToLLVM/ScanOpToLLVM.cpp

```diff
@@ -128,8 +128,8 @@ static void AddPartialReduce(SmallVector<SmallVector<Value>> &srcValues,
                              ConversionPatternRewriter &rewriter,
                              const TargetInfoBase &targetInfo,
                              ScanLoweringHelper &helper,
-                             SmallVector<Value> smemBases,
-                             SmallVector<Type> smemTypes, Value warpId,
+                             const SmallVector<Value> &smemBases,
```

Comment:
Coverity suggests to use ` std::move(smemBases)` when calling this function, but it looks like `smemBases` (as well as `smemTypes`) doesn't change inside the function, so I decided to change the signature.

Reply:
Can you make them `ArrayRef`?

Reply:
Done, thanks for the review @Jokeren!